### PR TITLE
Update .htaccess to include universal redirection rules

### PR DESCRIPTION
**Pull request by particthistle**
Adding universal regex redirection rules to allow for all old URLS to automatically redirect to new versions of their articles.

@conconnl please merge #22 first. I'll need to then update this one and recommit.

I've tested these on one of my sites against a large sample of the list of URLs to be redirected, and it's worked on my server for all cases.

### CODE REVIEW

**Comment by conconnl** [comment]
Done

**Comment by particthistle** [comment]
@conconnl I've added the Yandex commit to this request now.

Happy for you to merge, or wait for Magazine team to give it some extra consideration.

File: www/.htaccess [hunk 1]
```diff
@@ -59,6 +59,59 @@ RewriteCond %{REQUEST_URI} ^/topics/itemlist/(.*)  [OR]
...
+# ===== Redirect international story URLs to tags =====
+RewriteRule ^international-stories-all/articles-in-(.*)-all$ https://magazine.joomla.org/all-issues/tags/$1 [L,R=301,NC]
+RewriteRule ^faq/(.*)$ https://magazine.joomla.org/faq [L,R=301,NC]
+
+# ===== Other Miscellaneous URLS not covered by rules above =====
+RewriteRule ^issues/issue-july-2013/itemlist/tag/Google%20Summer%20of%20Code$ https://magazine.joomla.org/all-issues/tags/Google%20Summer%20of%20Code [L,R=301,NC]
+RewriteRule ^item/3289-episode-iv-a-new-user-interface-for-the-joomla-backend$ https://magazine.joomla.org/all-issues/november-2017/episode-iv-a-new-user-interface-for-the-joomla-backend [L,R=301,NC]
+RewriteRule ^item/3305-a-new-visual-language-for-joomla-4$ https://magazine.joomla.org/all-issues/february-2018/a-new-visual-language-for-joomla-4  [L,R=301,NC]
+RewriteRule ^subscribe$ https://magazine.joomla.org/magazine-newsletter  [L,R=301,NC]
+RewriteRule ^$ https://magazine.joomla.org/  [L,R=301,NC]
```
Comment (JazParkyn) [inline www/.htaccess]: @particthistle can you check this line? It appears to be redundant

Comment (particthistle) [reply www/.htaccess]: @JazParkyn - less redundant and more extremely greedy, and as a result wrong. It actually takes any other URL and redirects it back to the magazine home page 😦

**Review by particthistle — COMMENTED** [review]
Removing line
`RewriteRule ^$ https://magazine.joomla.org/  [L,R=301,NC]`

**Comment by conconnl** [comment]
@JazParkyn is this correct now, so I can merge it?

**Comment by JazParkyn** [comment]
Looks good to me

**Review by JazParkyn — APPROVED** [review]

**Pull request merged**